### DISCUSSION TITLE
Local reusable workflows

### DISCUSSION THREAD

**Discussion by mastercoms** [nektos/act#1505] [Q&A] [answered]
I understand it may be difficult to support full reusable workflows, but is there a plan to support local reusable workflows separately or is there just difficulty in implementing reusable workflows in general? I would have commented on the issue but it's locked understandably.

https://github.blog/changelog/2022-01-25-github-actions-reusable-workflows-can-be-referenced-locally/

**Comment by KnisterPeter** [comment, marked as answer]
Have a look at this: https://github.com/nektos/act/pull/1423
